online-shop/src/components/Product.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import TableRow from '@material-ui/core/TableRow';
+import TableBody from '@material-ui/core/TableBody';
+import Button from '@material-ui/core/Button';
+import {Route, Link, Switch} from 'react-router-dom';
+import ProductDetails from './ProductDetails';
+import { Theme } from '@material-ui/core/styles/createMuiTheme';
+import withStyles from '@material-ui/core/styles/withStyles';
+import createStyles from '@material-ui/core/styles/createStyles';
+import TableCell from '@material-ui/core/TableCell/TableCell';
+
+const MyCellType = withStyles((theme: Theme) =>
+  createStyles({
+    body: {
+      fontSize: 25,
+    }
+  }),
+)(TableCell);
+
+function Product(props: { id: string, name: string; category: string, price: number, details: string; }) {
+    return (
+        <TableBody>
+            <TableRow>
+                <MyCellType style={{fontSize: 20}} align="left">{props.name}</MyCellType>
```

Comment:
Stiluri in fisiere de css. Si pentru liniile urmatoare.

Reply:
Exact cu asta m-am chinuit ieri cand ati venit sa vedeti proiectul meu. Daca de exemplu aici nu folosesc _css inline_, sau altundeva _!important_ in fisierul css, nu pot nicicum controla acel property... Are ceva de facut cu Material ui, dar nu stiu cum sa rezolv altfel.